### ISSUE TITLE
Map Nunchuck Stick and Wiimote arrow pad seperatly

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dborth/snes9xgx#253]
```
Without a gamepad available, I had a control configuration for the Wiimote
+ Nunchuck that seemed pretty close the the original SNES pad, using the
Nunchuck in my left hand as the arrow buttons, and the arrows on the
Wiimote as my A/B/X/Y buttons, and the Z/B as right and left shoulder.

Unfortunately, it seems that the joystick and Wiimote arrow buttons are
mapped as the same keys, so this does not seem possible to configure it as
I had hoped.

Long story short, is it possible to split up the Nunchuck joystick and
Wiimote arrow buttons as separate inputs?
```

Original issue reported on code.google.com by `mdszep...@gmail.com` on 10 Jun 2009 at 12:51
